test/core/OptionsPremiumPricer.ts

```diff
@@ -235,8 +235,8 @@ describe("OptionsPremiumPricer", () => {
       );
 
       assert.isBelow(
-        parseInt(math.wmul(premiumCall, underlyingPriceShifted).toString()),
-        parseInt(premiumPut.toString())
+        parseInt(premiumPut.toString()),
+        parseInt(math.wmul(premiumCall, underlyingPriceShifted).toString())
```

Comment:
This was failing - I think you got the order wrong before

Reply:
weird, was working locally. and it passed cci because I merged it. going from usd -> eth -> usd has a little bit of rounding imprecision which at end makes it less than original usd and therefore less than put (at block usd <-> usdc is same)